drizzle.config.ts

```
@@ -1,12 +1,19 @@
 import 'dotenv/config'
 import { defineConfig } from 'drizzle-kit'
 
+const dbCredentials =
+  process.env.NODE_ENV === 'test'
+    ? {
+        url: 'file:test.db',
+      }
+    : {
+        url: process.env.TURSO_DATABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)